atlp-rwanda/hackers-ec-Fe
# #feat-buyer-add-wishlist-187419026: buyer can save product in wishlist

##what does this PR do
it implement how buyer can save and delete product in wishlist 

## how to test this PR
1. Login with buyer account 
2.  Checkout to feat-buyer-add-wishlist-187419026 and run npm install
3. Run npm run dev to start the application locally
4. add product to wishlist 
5. Click My wishlist in Navbar and see all wishlist information,
